### PR TITLE
Feat home page 02

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
1. Added TopNav, SideNav, HomeBanner components
2. Completed styling the Homepage

Screenshot (124)

### CODE REVIEW

File: src/components/HomeBanner/HomeBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Fade } from "react-slideshow-image";
+import React from "react";
+import "./HomeBanner.css";
+import "react-slideshow-image/dist/styles.css";
+import HomeImg1 from "../../assets/HomeImg1.jpg";
+import HomeImg2 from "../../assets/HomeImg2.jpg";
+import HomeImg3 from "../../assets/HomeImg3.jpg";
+import HomeImg4 from "../../assets/HomeImg4.png";
+import HomeImg5 from "../../assets/HomeImg5.jpg";
```
Comment (nishasen) [inline src/components/HomeBanner/HomeBanner.jsx:9]: you can keep this in a separate file and call the array here

**Review by nishasen — APPROVED** [review]
Everything looks good, go ahead

File: src/components/HomeBanner/HomeBanner.jsx (hunk 1 above)
Comment (vaibhav-mahalle) [reply src/components/HomeBanner/HomeBanner.jsx:9]: will keep that in mind

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
 import "./App.css";
 import logo from "./logo.png";
-import {RouterPath} from "./routes/RouterPath";
-import { ToastContainer } from 'react-toastify';
-import 'react-toastify/dist/ReactToastify.css';
+import { RouterPath } from "./routes/RouterPath";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```
Comment (srejitk) [inline src/App.js:5]: Try giving react hot toast a try. It'll suit your theme better in my opinion.

File: src/components/HomeBanner/HomeBanner.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+.hero-container{
+    height: 65vh;
+    margin: 0rem auto;
+    width: 60vw;
```
Comment (srejitk) [inline src/components/HomeBanner/HomeBanner.css:4]: Refrain from using vh and vw as these might cause issues when you aim for responsiveness later on.

File: src/components/HomeBanner/HomeBanner.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    object-fit: fill;
+}
+
+@media screen and (max-width: 760px) {
+    .hero{
+        object-fit: cover;
+    }
+
+    .hero-container{
+        width: 350px;
```
Comment (srejitk) [inline src/components/HomeBanner/HomeBanner.css:25]: Use relative units got this.

File: src/components/HomeBanner/HomeBanner.jsx (hunk 1 above)
Comment (srejitk) [inline src/components/HomeBanner/HomeBanner.jsx:9]: Save these paths in an object/array and then you can call them here in a single import line

File: src/components/TopNav/TopNav.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+function TopNav() {
+  const { authState, authDispatch } = useAuth();
+  const { isLoggedIn } = authState;
+  const navigate = useNavigate();
+  return (
+    <header className="nav-container">
+      <div>
+        <AiOutlineMenu size={35} />
+        {/* <img src={BrandLogo} alt="Brand-logo" /> */}
```
Comment (srejitk) [inline src/components/TopNav/TopNav.jsx:18]: Remove this comment if not required.

**Review by srejitk — APPROVED** [review]
Good work with the code. Like how you've approached it all in a modular way. A few minor suggestions provided.

**Review by shubhamkumar648 — APPROVED** [review]
- ui looks good
-  sreejith already left coment  of required changes 
- remaming look good to merge

File: src/components/HomeBanner/HomeBanner.css (hunk 3 above)
Comment (vaibhav-mahalle) [reply src/components/HomeBanner/HomeBanner.css:4]: what alternate approach do you suggest

File: src/components/HomeBanner/HomeBanner.css (hunk 4 above)
Comment (vaibhav-mahalle) [reply src/components/HomeBanner/HomeBanner.css:25]: will make the necessary changes

File: src/components/TopNav/TopNav.jsx (hunk 5 above)
Comment (vaibhav-mahalle) [reply src/components/TopNav/TopNav.jsx:18]: it is useful for me to insert brand logo over there.

File: src/components/HomeBanner/HomeBanner.jsx (hunk 1 above)
Comment (vaibhav-mahalle) [reply src/components/HomeBanner/HomeBanner.jsx:9]: will make the necessary changes

File: src/App.js (hunk 2 above)
Comment (vaibhav-mahalle) [reply src/App.js:5]: thanks for the suggestion will think about it after adding the functionalities.

**Pull request merged**